vyper/semantics/validation/local.py

```diff
@@ -186,9 +186,7 @@ def visit(self, node):
     def visit_Attribute(self, node):
         if node.get("value.id") == "msg" and node.attr == "data":
             parent = node.get_ancestor()
-            is_slice = parent.get("func.id") == "slice"
-            is_len = parent.get("func.id") == "len"
-            if not is_slice and not is_len:
+            if parent.get("func.id") not in ["slice", "len"]:
```

Comment:
nit: please use a tuple here
```suggestion
            if parent.get("func.id") not in ("slice", "len"):
```